uuid-ballerina/tests/uuid-negative-test.bal

```diff
@@ -18,41 +18,76 @@ import ballerina/test;
 
 @test:Config {}
 isolated function testCreateType3AsStringNegative() {
-    test:assertEquals(createType3AsString(NAME_SPACE_DNS, " "), error Error("Name cannot be empty"));
+    string|Error uuid3 = createType3AsString(NAME_SPACE_DNS, " ");
+    if (uuid3 is string) {
+        test:assertFail("function did not return an error when the name is empty");
+    }
```

Comment:
Don't we need to validate the Error message here?

Reply:
Updated in d4f4129